editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -63,6 +63,21 @@ void EditorAssetLibraryItem::configure(const String &p_title, int p_asset_id, co
 	price->set_text(p_cost);
 }
 
+void EditorAssetLibraryItem::clamp_width(int p_max_width) {
+	int text_pixel_width = title->get_theme_default_font().ptr()->get_string_size(title->get_text()).x * EDSCALE;
+	if (text_pixel_width > p_max_width) {
+		// Truncate title text to within the current column width.
+		String full_text = title->get_text();
+		title->set_tooltip_text(full_text);
```

Comment:
I think we can just always set the tooptip, it never hurts.

Reply:
I moved these two lines above the conditional. Thanks!